products/compute/terraform.yaml

```diff
@@ -1156,6 +1156,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       id: !ruby/object:Overrides::Terraform::PropertyOverride
         name: proxyId
+  TargetInstance: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "target_instance_basic"
+        primary_resource_id: "default"
+        vars:
+          target_name: "target"
+          instance_name: "target-vm"
+    properties:
+      zone: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+        required: false # the provider-default value will be used if not specified
+      instance:  !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'compareSelfLinkOrResourceName'
+        custom_expand: 'templates/terraform/custom_expand/self_link_from_name.erb'
+        description: |
+          The Compute instance VM handling traffic for this target instance.
+          Accepts the instance self-link or name.
```

Comment:
How is zone determined when it's not specified with a partial or full URI?

Reply:
There's a zone parameter. Currently the zone is technically for the target, but I don't think they'll change it to allow a different zone, and in that case the user can still use the projects/ level path or self-link. I'll add the projects-level relative path to the docs (since it's supported by the diff suppress), but mostly the API allows `zones/X/instances/X` and I don't really want to add back my special diff suppress just to support that. Mostly I selfishly don't want to encourange TF users to use that format.

Reply:
Do you mind detailing where the name / project come from if unspecified in the docs themselves?

Reply:
👍